server/test/controllers/FileControllerTest.java

```diff
@@ -164,6 +171,21 @@ public void acledAdminShow_globalAdmin_returnsUnauthorizedResult() {
     assertThat(result.status()).isEqualTo(UNAUTHORIZED);
   }
 
+  @Test
+  public void acledAdminShow_globalAdminWithPrivledges_redirects() {
+    when(mockSettingsManifest.getAllowCiviformAdminAccessPrograms(request)).thenReturn(true);
+
+    ProgramModel program = ProgramBuilder.newDraftProgram().build();
+    createProgramAdminWithMockedProfile(program);
+    String fileKey = fakeFileKey(1L, program.id);
+    createStoredFileWithProgramAccess(fileKey, program);
+    String encodedFileKey = encodefakeFileKey(fileKey);
+
+    Result result = controller.acledAdminShow(request, encodedFileKey);
+
+    assertThat(result.status()).isEqualTo(SEE_OTHER);
+  }
+
```

Comment:
(blocking) We should probably have a test that mocks the return value to false and not rely on the config defaults. I'm not sure if that should be a whole separate test or just explicitly setting `acledAdminShow_redirects` with the mock. What do you think?

Reply:
Updated the return value to false by default (unless otherwise specified). Let me know what you think.

Reply:
Added a few other tests with the global admin role as well.